webpack.config.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    bundle: './src/index.js',
+  },
+  devServer: {
+    static: './dist',
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    assetModuleFilename: 'images/[name][ext]',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(s[ac]|c)ss$/i,
```

Comment:
Kindly enable eslint for it is not allowed to disable linters. linters help us write clean code therefore it is against microverse policy to disable linters.